frontend/src/audio.html

```diff
@@ -0,0 +1,84 @@
+---
+pagination:
+  data: audios
+  size: 1
+  alias: audio
+permalink: "/audios/{{ audio.objectID | slug }}/"
+layout: base
+templateEngineOverride: md, njk, html
+eleventyComputed:
+  title: "{{ audio.objectID }}"
+---
+<section class="flex flex-col m-10">
+  <div class="flex flex-col">
+    <div class="flex align-baseline justify-between">
+      <div>
+        <p class="font-bold text-lg">{{ audio.title }}</p>
+      </div>
```

Comment:
Looks like unnecessary nesting. `p` itself can be a flex item.

Reply:
It can be even `span` instead of `p`. For text elements `span` is better semantically.

Reply:
removed nesting with span tag